cypress/integration/features/teams/createEditDeleteTeam.feature

```diff
@@ -0,0 +1,41 @@
+Feature: CRUD operation on Teams
+
+  As a teacher I want to create/edit/delete the team so that I can manage the team.
+
+Scenario: Creating a new team by Teacher
+  Given I am logged in as a 'teacher' at 'brb'
+  When I go to teams overview
+  When I click on add team button
+  Then I see new team creation page
+  When I enter in the title 'this is a team'
+  When I enter in the description 'this is team description'
+  Then I see team chat activation check box is visible
```

Comment:
Question: Related to RC and Video conference checkbox option. How do we know that RC/Video conference is activated? I did not see any step other than this in which team was created by marking on the checkbox and no other user (student/admin) or same user in some other step see the side panel in which RC is opened? e.g

```
Then I see team chat activation check box is visible
...
Then I see RC in the side panel
```